satpy/etc/readers/aws_l1b_nc.yaml

```diff
@@ -0,0 +1,425 @@
+reader:
+  name: aws_l1b_nc
+  short_name: AWS L1B RAD NetCDF4
+  long_name: AWS L1B Radiance (NetCDF4)
+  description: Reader for the ESA AWS (Arctic Weather Satellite) Sounder level-1b files in netCDF4.
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [aws,]
+  status: Beta
+  default_channels: []
```

Comment:
you could also add the remote reading status of the reader.

Reply:
I don't think `default_channels` is used so it could be removed.

Reply:
good point, thanks

Reply:
Fixed in [f039715](https://github.com/pytroll/satpy/pull/2565/commits/f03971533795e871342299e33fa5536dd88380b1)